cdap-ui/app/features/tracker/controllers/metadata-ctrl.js

```diff
@@ -15,13 +15,17 @@
  */
 
 class TrackerMetadataController{
```

Comment:
should start to have the practice of documenting what a class is responsible for.